proto/blob/v2/blob.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
-package proto.blob.v1;
+package proto.blob.v2;
```

Comment:
[optional] if we have to increment this for every future go-release version because node will import the latest version and the trailing version, then maybe this version can match the go-square version. In other words

```suggestion
package proto.blob.v3;
```

Reply:
Yeah I thought about that afterwards and then didn't get around to implementing. It would be a small consolation to have it match the square version - just to keep something simpler - but perhaps we can do that the next major release